src/controllers/order/index.ts

```diff
@@ -20,6 +38,66 @@ export class order extends Controller {
     this.db = new Database()
   }
 
+  /**
+   * Returns the details of all order demands.
+   * @summary List all order demands
+   */
+  @Get('/')
+  public async getAll(): Promise<DemandResponse[]> {
+    const capacities = await this.db.getDemands(DemandSubtype.order)
+    const result = await Promise.all(
+      capacities.map(async (order: DemandResponse) => ({
```

Comment:
capacity from copy paste?

Reply:
updated

Reply:
Looks like one of the unstagged changes. 
<img width="694" alt="Screenshot 2023-04-03 at 13 19 26" src="https://user-images.githubusercontent.com/11794402/229507184-ee4b88df-237f-409b-acd8-ea95c0210202.png">

Please confirm